template/package.json

```
@@ -18,7 +18,7 @@
     {{#router}}
     "vue-router": "^3.0.1",
     {{/router}}
-    "bootstrap": "^4.0.0-beta.2",
+    "bootstrap": "^4.3.1",
     "bootstrap-vue": "^1.2.0",
```

This line should also be updated as well:

```suggestion
    "bootstrap-vue": "^2.0.0-rc.16",
```